src/modals/manage-account/manage-account.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@angular/core';
+import { NavController, NavParams, ViewController } from 'ionic-angular';
+import { Validators, FormBuilder, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'manage-account-modal',
+  templateUrl: 'manage-account.html',
+})
+export class ManageAccountModal {
+  private accountChangeForm : FormGroup;
+  
+  constructor(
+      public navCtrl: NavController, 
+      public navParams: NavParams, 
+      public viewCtrl: ViewController,
+      private formBuilder: FormBuilder) {
+        this.accountChangeForm = this.formBuilder.group({
+            firstName: [''],
```

Comment:
this is where we can leverage the power of formBuilder's built in authentication tools